src/components/periods/PeriodSelect.js

```diff
@@ -89,15 +89,12 @@ class PeriodSelect extends Component {
         }
 
         return (
-            <div style={{ height: 100, ...style }}>
+            <div className={className} style={{ height: 100 }}>
```

Comment:
Can the `height` style also be defined in the style file?

Reply:
> Can the `height` style also be defined in the style file?

Yes will fix in a separate PR.